www/src/pages/playground.tsx

```diff
@@ -35,20 +46,54 @@ export default function Playground({ location }) {
     return () => clearInterval(iframeUrlPoll);
   }, []);
 
+  useEffect(() => {
+    setCopyUrlState('default');
+  }, [location.href]);
+
   return (
     <div className="d-flex flex-column w-100 vh-100 m-0 p-0">
       <SEO title="Playground" />
       <div className="pgn-doc__header py-3 bg-dark text-white sticky-top">
-        <div className="d-flex align-items-center justify-content-center">
+        <div className="d-flex align-items-center justify-content-end px-4" style={{ gap: '16px' }}>
```

Comment:
Maybe for the Playroom header good solution use the Stack component. What do you think?